tcms/static/js/bugs.js

```diff
@@ -0,0 +1,49 @@
+const bugDetailsCache = {}
+
+function loadBugs(selector, filter) {
+    $(selector).DataTable({
+        ajax: (data, callback, settings) => {
+            dataTableJsonRPC('TestExecution.get_links', filter, callback);
+        },
+        columns: [
+            {
+                data: null,
+                render: (data, type, full, meta) => `<a href="${data.url}" class="bug-url">${data.url}</a>`
+            },
+            {
+                data: null,
+                render: function (data, type, full, meta) {
+                    return `<a href="#bugs" data-toggle="popover" data-html="true"
+                        data-content="undefined" data-trigger="focus" data-placement="top">
+                        <span class="fa fa-info-circle"></span>
+                        </a>`;
+                }
+            },
+        ],
+        dom: "t",
+        language: {
+            loadingRecords: '<div class="spinner spinner-lg"></div>',
+            processing: '<div class="spinner spinner-lg"></div>',
+            zeroRecords: "No records found"
```

Comment:
We need this string to be placed in the HTML template so it can be translated. You can use a `<template>` tag and just select it's text for example.